src/client.cpp

```diff
@@ -201,6 +206,17 @@ ADDON_STATUS ADDON_SetSetting(const char *settingName, const void *settingValue)
       return ADDON_STATUS_NEED_RESTART;
     }
   }
+  
+  if (name == "xmlTVFile")
+  {
+    string xmlTVFileProp = (const char*) settingValue;
```

Comment:
It is not a good idea (and  not common  practice) to omit the std namespace. This readily can lead to hard to catch errors.

In  c++, we have better cast operators than Java. You should use always use  c++ casts for reasons described a million times in  the net. ;-)